bin/dartdoc.dart

```diff
@@ -44,6 +44,16 @@ void main(List<String> arguments) {
     exit(1);
   }
 
+  var inputDir;
+  var input_dir = args['input-dir'];
```

Comment:
what's the difference between this one and inputDir? Are there more descriptive variable names?